hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiDeploymentBaseTest.java

```diff
@@ -539,6 +540,133 @@ public void deplomentAutoForceAction() throws Exception {
         assertThat(actionStatusMessage.getStatus()).isEqualTo(Status.RETRIEVED);
     }
 
+    @Test
+    @Description("Attempt/soft deployment to a controller. Checks if the resource response payload for a given deployment is as expected.")
+    public void deploymentDownloadOnlyAction () throws Exception {
```

Comment:
this test is very long, maybe the test setup could be refactored into separate method(s)?

Reply:
Will do this in another PR